crates/sui-types/src/error.rs

```diff
@@ -491,6 +500,14 @@ impl std::convert::From<&str> for SuiError {
     }
 }
 
+impl SuiError {
+    pub fn indicates_epoch_change(&self) -> bool {
+        let err_str = self.to_string();
+        err_str.contains(VALIDATOR_HALTED_ERROR_MSG)
```

Comment:
When VALIDATOR_HALTED_ERROR_MSG is seen, epoch change hasn't finished yet. Reconfig at this point will fail. Is that intended?

Reply:
My intuition is that eventually (doesn't need to be in this PR), the quorum driver should be able to detect that a quorum of validators are halted, and block wait there with some timeout while processing transactions. Doing so the quorum driver can hide the errors from any client.
From loadgen's point of view (which should be a client of quorum driver), should only worry about actual epoch changes and reconfig the authority aggregator. (and similarly, when a fullnode detects epoch changes, it also reconfigs the authority aggreagator inside quorum driver)

Reply:
after some thoughts I think we should make it sleep for a bit, no need to rush

Reply:
So the thing is that since you are relying on the system object to get the committee, it will only update at the new epoch (the first transaction of the new epoch). This means that you will not be able to get anything meaningful until the epoch change has fully finished.
For this reason, I think there is no reason to react to the halt error here in the load gene. You can just rely on the missing committee error.

Reply:
In the latest commit, before getting down with doing reconfig, the task sleeps for a few seconds (10 ~ 20 s at the moment). This should be long enough time for validators to unhalt and hop into the next epoch. 

I'm not so sure about how quorum driver should behave when it detects a potential epoch change. It could reactively wait until the fullnode configs itself (through node sync, possibly) and then its AuthorityAggregator will be updated too. But agree this is a separate discussion topic.